Makefile

```diff
@@ -1,12 +1,26 @@
 # cfs-gui Makefile
 
 phony := build 
+phony += all
+phony += backend
+phony += frontend
 phony += clean
 
+default: build
+
 build: build.sh
-	sh build.sh
+	bash build.sh --all
+
+all:
+	bash build.sh --all
+
+backend:
+	bash build.sh --backend
+
+frontend:
+	bash build.sh --frontend
 
 clean: bin
-	rm -rf bin
+	bash -rf bin
```

Comment:
Typo `bash -rf bin ` should be `rm -rf bin `

Reply:
done